.pipelines/templates/template-prod-e2e-steps.yml

```diff
@@ -1,29 +1,59 @@
 parameters:
+  aroVersionStorageAccount: ''
+  azureDevOpsE2EJSONSPN: ''
   location: ''
+  rpMode: ''
   subscription: ''
-  azureDevOpsE2EJSONSPN: ''
 
 steps:
-# TODO(mj) we need to checkout code to run tooling like kubeconfig. This should go away once we refactor everything into golang.
-- template: ./template-checkout.yml
-- template: ./template-az-cli-login.yml
-  parameters:
-    azureDevOpsJSONSPN: ${{ parameters.azureDevOpsE2EJSONSPN }}
 - script: |
     set -e
     export LOCATION=${{ parameters.location }}
     export AZURE_SUBSCRIPTION_ID=${{ parameters.subscription }}
 
-    trap 'rm -f devops-spn.json' EXIT
+    trap 'rm -f devops-spn.json ; for c in $(docker ps -aq); do docker rm -f $c; done; docker image prune -af; rm -rf ~/.docker/config.json' EXIT
```

Comment:
prefixing a `set +e; ` on all traps would be good.  Why not `docker logout -a` instead of the rm -rf ?

Reply:
There is not `-a` flag in docker. it is podman flag only.

Reply:
ok